site/content/en/docs/Examples/xonotic.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Deploying and Running Xonotic Server Using Agones"
+linkTitle: "Xonotic"
+date:
+publishDate:
+description: >
+  This Xonotic example shows setting up, deploying, and managing a Xonotic game server on a Kubernetes cluster with Agones. It uses a simple Go wrapper to connect existing game servers with Agones, making it straightforward to run games in the cloud.
+---
+
+## Prerequisite
+
+ To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+
+  - Agones is installed on your cluster. Refer to the [Agones guide](https://agones.dev/site/docs/installation/install-agones/).
+
+  - The Xonotic client downloaded for gameplay. Download it from [Xonotic](http://www.xonotic.org)
```

Comment:
Don't we also need the code for the example? 😃

Should use the `ghlink` shortcode to do this, since it will link to the appropriate release branch.

(Wondering if this is a missing prerequisite from the other example pages? 🤔 )

Reply:
I have added the Xonotic code link to the prerequisite.
Will include the code link in the other two submitted PRs.